### ISSUE TITLE
Resolve discrepancies with the IRS

### ISSUE THREAD

**Issue by chadwhitacre** [gratipay/inside.gratipay.com#1220]
@JessWhit and I are working through a [CP2000](https://www.irs.gov/individuals/understanding-your-cp2000-notice) from the IRS for 2015 that we received in September, 2017. Basically we should've reported total processing volume as income and then crammed our escrow liability into "Other Expense" (plus reporting our processing fee expense, i.e., the fee pass-through), rather than reporting donations-to-Gratipay-via-Gratipay as our top-line income (and leaving processing fees out entirely). As it was, the IRS took it that our Braintree 1099-K showing $36,428.04 in volume represented direct income to Gratipay vs. the $18,758 that we reported.

In fact, we processed an additional $142,239 on Balanced that the IRS does not seem to know about (2015 is the year Balanced went out of business, and I don't find that we received a 1099-K from them).

Sooooooo back in September we met with a CPA who went to a local IRS office with us and advised us on how to file. We mailed our CP2000 response with a revised 1040 in November. Unfortunately, while the Service did cash our check (for $568 vs. the $9,220 they were proposing), they mistakenly failed to "post" our response. So we got an escalation notice in January. We called, mailed our response a second time, and then yesterday we called again. Our response did post this time, on Feb 2, and we are expecting to receive their response to our proposed revision via mail in mid-March.

Once we sort out 2015 I guess we will have to wait a year to see what happens with 2016.

We are in the process of preparing 2017 on the pattern of our 2015 revision. Ideally we reach agreement with the IRS re: 2015 in time to confidently file 2017. Otherwise this may drag out even further (i.e., into 2019) if we also have to let a 2017 review run its course.

**Comment by chadwhitacre** [comment]
Called and talked to the IRS again. They've received the paperwork we sent in January and "it's in a batch of work that hasn't been assigned yet," but as long as they have paperwork in the queue then we shouldn't be moved into assessment until the paperwork has been reviewed. She said call back in three weeks to check again.

**Comment by chadwhitacre** [comment]
Good news!

> Thank you for your response to our previous notice. Based on the information you provided, we've determined you owe $37 (including interest).

tl;dr they basically accepted our revisions* 💃, and the $37 is interest from April 2015 until November 2017 when they received our $568 check.

So now we wait for a bill for $37 and pay it when we get it ... and wait for 2016! 🙄

--------

<i>\* It looks like they redid the numbers again but made it come out to exactly $568 to match what we gave them.</i> ¯\\\_(ツ)\_/¯

**Comment by chadwhitacre** [comment]
We received the CP2000 for 2015 in the spring of 2018. We're well past the spring of 2019, so I think we should assume that we're not going to receive a CP2000 for 2016. We can always reopen if/when we do.

I also assume we paid the $37 bill by now, or that we aren't going to get it. :)
